src/main/scala/utils/CommandLineArgsParser.scala

```diff
@@ -0,0 +1,21 @@
+package utils
+
+/**
+  * Helper for parsing command line args
+  *
+  * Parse -d <folder_path> to List(("-d", "<folder_path>"))
+  */
+object CommandLineArgsParser {
+  /**
+    * Get Array of args and parse it to List of Tuple2 (pairs)
+    *
+    * @param args
+    * @return
+    */
+  def parse(args: Array[String]): List[(String, String)] = {
+    args.iterator
+      .sliding(2, 2)
+      .withPartial(false).toList
+      .map(_ match { case List(a, b) => (a,b) })
```

Comment:
could be just `.map(case List(a, b) => a, b)`

Reply:
Do not work for me. It throws exception while compiling. Maybe because of Scala version. Need to be investigated.

Also refactored main class ( remove aggregation logic to separate class and fix issue with getting all crimes instead of thefts only. )

Reply:
@piontkivskyi damn, my bad, i missed braces: should be `.map{case List(a, b) => (a, b)}`

Reply:
@azhur fixed, thanks.